include/aws/common/byte_buf.h

```diff
@@ -51,6 +51,10 @@ static inline struct aws_byte_cursor aws_byte_cursor_from_buf(const struct aws_b
     return (struct aws_byte_cursor){.ptr = buf->buffer, .len = buf->len};
 }
 
+static inline struct aws_byte_cursor aws_byte_cursor_from_array(const uint8_t c_str[], size_t len) {
```

Comment:
Style: I prefer `uint8_t *` in arguments; array syntax comes with some conceptual baggage that doesn't apply in argument lists.

Reply:
I agree 100%. I always write it that way when doing my own code, but I had gone the other way to match the style of aws_byte_buf_from_array in the same file. I will change both of them to match the style you requested.

Incidentally, I prefer to keep spaces on both sides of the * as well when I am declaring a pointer, but it looks like I am losing that battle in this code.